src/map/utils/attackutils.cpp

```diff
@@ -151,15 +151,32 @@ namespace attackutils
         {
             return damage;
         }
+
         uint32 originalDamage = damage;
-        auto occ_extra_dmg = battleutils::GetScaledItemModifier(PChar, PWeapon, Mod::OCC_DO_EXTRA_DMG);
-        auto occ_extra_dmg_chance = battleutils::GetScaledItemModifier(PChar, PWeapon, Mod::EXTRA_DMG_CHANCE);
-        if (occ_extra_dmg > 0 && occ_extra_dmg_chance > 0)
+        int16 occ_do_triple_dmg = PChar->getMod(Mod::REM_OCC_DO_TRIPLE_DMG) / 10;
+        int16 occ_do_double_dmg = PChar->getMod(Mod::REM_OCC_DO_DOUBLE_DMG) / 10;
+        float occ_extra_dmg = battleutils::GetScaledItemModifier(PChar, PWeapon, Mod::OCC_DO_EXTRA_DMG) / 100.f;
```

Comment:
there needs to be a ranged version of occ do extra damage, otherwise getting empyrean aftermath on a melee weapon would give the double/triple damage to ranged attacks

also, should it be mainhand only as well?